assets/js/index.js

```diff
@@ -0,0 +1,92 @@
+// TODO: Include packages needed for this application
+const inquirer = require("inquirer");
+const fs = require("fs");
+const generateMarkdown = require("../utils/generateMarkdown");
+
+// TODO: Create an array of questions for user input:
+// questions declared outside of the inquirerAsync function (-> readability)
```

Comment:
great point! but avoid keep these comments in your code

Reply:
Done